website/docs/docs/build/metrics-overview.md

```diff
@@ -51,7 +56,7 @@ metrics:
     type: conversion # Required
     label: # Required
     type_params: # Required
-      fills_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
+      fill_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
```

Comment:
This isn't the correct schema. Can we match the one in the conversion metric docs? 
```

 - name: The metric name # Required
    description: The metric description # Optional
    type: conversion # Required
    label: YOUR_LABEL # Required
    type_params: # Required
      conversion_type_params: # Required
        entity: ENTITY # Required
        calculation: CALCULATION_TYPE # Optional. default: conversion_rate. options: conversions(buys) or conversion_rate (buys/visits), and more to come.
        base_measure: 
          name: The name of the measure # Required
          fill_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
          join_to_timespine: Boolean that indicates if the aggregated measure should be joined to the time spine table to fill in missing dates. Default `false`. # Optional
        conversion_measure:
          name: The name of the measure # Required
          fill_nulls_with: Set the value in your metric definition instead of null (such as zero) # Optional
          join_to_timespine: Boolean that indicates if the aggregated measure should be joined to the time spine table to fill in missing dates. Default `false`. # Optional
        window: TIME_WINDOW # Optional. default: infinity. window to join the two events. Follows a similar format as time windows elsewhere (such as 7 days)
        constant_properties: # Optional. List of constant properties default: None
          - base_property: DIMENSION or ENTITY # Required. A reference to a 
```

Reply:
got it and double checked teh rest of the other metric types!